nginx/nginx_test.go

```diff
@@ -165,7 +157,10 @@ func TestFailsIfNginxDiesEarly(t *testing.T) {
 	lb := newUpdaterWithBinary(tmpDir, "./fake_failing_nginx.sh")
 
 	assert.Error(lb.Start())
```

Comment:
I wonder why this generates an error. Shouldn't `Start` succeed, since it doesn't start nginx yet?